README.md

```diff
@@ -106,3 +112,5 @@ Then to stop the container type `docker stop hamper`. To start it back up type
 Refer to the [docker docs][docker] for more usage details.
 
 [docker]: http://docs.docker.io/en/latest/
+[dburl]: http://www.sqlalchemy.org/docs/core/engines.html#sqlalchemy.create_engine
+[trial]: http://twistedmatrix.com/trac/wiki/TwistedTrial
```

Comment:
Usually I like to put the link specs in the same section as the where the link was used, but it doesn't really matter.